### PR TITLE
Update containerd version in workflows

### PR DESCRIPTION
**Pull request by sondavidb**
**Issue #, if available:**
Adjacently related to #1392

**Description of changes:**
Updated containerd version in CI, which should help tests pass in #1392 

**Testing performed:**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by austinvazquez — APPROVED** [review]
LGTM on CI pass, thanks.

**Comment by sondavidb** [comment]
Had to make a tiny edit to make the TestSnapshotterSystemdStartup test case to make it more consistent.

**Review by Shubhranshu153 — APPROVED** [review]

File: integration/startup_test.go [hunk 1]
```diff
@@ -104,12 +104,14 @@ func TestSnapshotterSystemdStartup(t *testing.T) {
...
 			defer cleanup()
+
+			sh.X("containerd", "--version")
 			// We're not using `rebootContainerd` here because that also restarts the soci-snapshotter
 			sh.Gox("containerd", "--log-level", containerdLogLevel)
 			tc.init(sh)
 			// 1s timeout is arbitrary so that the negative test doesn't take too long. It does mean that
 			// the snapshotter has to start up and prepare a snapshot within 1s in the socket activation case.
-			output, err := sh.CombinedOLog("ctr", "--timeout", "1s", "snapshot", "--snapshotter", "soci", "prepare", "test")
+			output, err := sh.CombinedOLog("ctr", "--timeout", "2s", "snapshot", "--snapshotter", "soci", "prepare", "test")
```
Comment (austinvazquez) [inline integration/startup_test.go:114]: nit: the comment no longer matches the behavior.

Comment (sondavidb) [reply integration/startup_test.go]: Thanks should be fixed now

**Review by Shubhranshu153 — APPROVED** [review]

**Pull request merged**